skynet/cli.py

```diff
@@ -75,11 +70,9 @@ def txt2img(*args, **kwargs):
 @click.option('--steps', '-s', default=26)
 @click.option('--seed', '-S', default=None)
 def img2img(model, prompt, input, output, strength, guidance, steps, seed):
-    from . import utils
+    from skynet.dgpu import utils
     config = load_skynet_toml()
-    hf_token = load_key(config, 'skynet.dgpu.hf_token')
-    hf_home = load_key(config, 'skynet.dgpu.hf_home')
-    set_hf_vars(hf_token, hf_home)
+    set_hf_vars(config.dgpu.hf_token, config.dgpu.hf_home)
```

Comment:
not a big deal but it'd be nice if there was at least some comments on what the `HF_` stands for 😉 

further in the future we might want to consider just naming this func something like `set_osenv(ns: dict)` and then be able to pass arbitrary runtime env vars to it.